sdk/python/setup.py

```diff
@@ -324,7 +324,7 @@ def run(self):
             self._generate_go_protos(f"feast/{sub_folder}/*.proto")
 
 
-class BuildGoEmbeddedCommand(build_py):
+class BuildGoEmbeddedCommand(Command):
```

Comment:
I didn't catch this earlier, what's the difference between build_py and command?